Code/Framework/AzCore/AzCore/Math/Vector3.inl

```diff
@@ -30,14 +32,28 @@ namespace AZ
         ;
     }
 
+    AZ_MATH_INLINE Vector3::Vector3(const Vector2& source)
+        : m_value(Simd::Vec3::FromVec2(source.GetSimdValue()))
+    {
+        m_z = 0.0f;
+    }
+
+    AZ_MATH_INLINE Vector3::Vector3(const Vector2& source, float z)
+        : m_value(Simd::Vec3::FromVec2(source.GetSimdValue()))
+    {
+        m_z = z;
```

Comment:
nit: `m_z` could still be in the initializer list for both

```c++
AZ_MATH_INLINE Vector3::Vector3(const Vector2& source, float z)
    : m_value(Simd::Vec3::FromVec2(source.GetSimdValue()))
    , m_z(z)
{
    ...
```
(repeated)

Reply:
you can't do that its already initialized from m_value

Reply:
Aaah interesting I see, cool please disregard this then :+1: